packages/fxa-payments-server/src/routes/Subscriptions/index.stories.tsx

```diff
@@ -1,18 +1,37 @@
-import React from 'react';
+import React, { ReactNode } from 'react';
 import { storiesOf } from '@storybook/react';
 import { action } from '@storybook/addon-actions';
 import MockApp, {
   defaultAppContextValue,
 } from '../../../.storybook/components/MockApp';
 import { SettingsLayout } from '../../components/AppLayout';
 import { Subscriptions, SubscriptionsProps } from './index';
+import {
+  PaymentUpdateStripeAPIs,
+  PaymentUpdateAuthServerAPIs,
+} from './PaymentUpdateFormV2';
 import { QueryParams } from '../../lib/types';
 import { APIError } from '../../lib/apiClient';
 import { FetchState } from '../../store/types';
 import { linkTo } from '@storybook/addon-links';
+import { CUSTOMER, FILTERED_SETUP_INTENT } from '../../lib/mock-data';
 
 function init() {
-  storiesOf('routes/Subscriptions', module)
+  setupVariantStories('routes/Subscriptions');
+  setupVariantStories('routes/SubscriptionsV2', {
```

Comment:
Managed to re-purpose all the existing Subscription stories for the V2 route with a bit of component wrapping that flips the `useSCAPaymentFlow` bit and plumbs in the API mocks. Couldn't really think of any unique stories here.